src/tests/integration/utils.py

```diff
@@ -1,5 +1,6 @@
 """Utility class and functions for integration testing"""
 import argparse
+from deepmerge import always_merger
```

Comment:
Looks like it's a third party package? If so, it should be imported in a separated group.